src/mappings.js

```diff
@@ -70,10 +70,12 @@ export const fieldsToIncludeFields = fields => {
 
   return fields
     .map(field => {
-      if (common.includes(field)) {
-        return field;
-      } else if (inputMappings.includes(field)) {
-        return mappings[field];
+      const lastField = field.substr(field.lastIndexOf('.') + 1);
```

Comment:
I would rename to `field` since we're not always dealing with a nested query.

We can probably make `field.substr(field.lastIndexOf('.') + 1);` more readable by rewriting to `field.split('.').slice(-1)[0];`

Reply:
You can make it even cleaner with:

```js
// Use array destructure to put the first item in the array
// in the lastField variable.
const [lastField] = field.split('.').slice(-1);
```